modules/core/keeper/keeper.go

```diff
@@ -63,7 +63,7 @@ func NewKeeper(
 		panic(fmt.Errorf("cannot initialize IBC keeper: empty scoped keeper"))
 	}
 
-	clientKeeper := clientkeeper.NewExtendedKeeper(cdc, key, paramSpace, stakingKeeper, upgradeKeeper)
+	clientKeeper := clientkeeper.NewKeeper(cdc, key, paramSpace, stakingKeeper, upgradeKeeper)
```

Comment:
Why change it back to `NewKeeper` other than `NewExtendedKeeper`?

Reply:
I found that we don't need to. In order to drop in our own IBC keeper that allows ZoneConcierge to hooks onto, we need to create an `ExtendedKeeper`, include the ZoneConcierge into the subscribers, and replace the original IBC client keeper with `ExtendedKeeper` (as done in https://github.com/babylonchain/babylon/pull/189/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R453-R463).

This makes our modifications less intrusive: our IBC-Go fork only adds two new files on extended keepers, adds two error types, and that's all.

Reply:
I see, so the extended Keeper is used by Babylon app. Thanks!